assets/src/edit-story/components/canvas/layout.js

```diff
@@ -133,28 +133,30 @@ const PageAreaSafeZone = styled.div`
   overflow: visible;
   position: relative;
   margin: auto 0;
-`;
-
-const PageAreaDangerZone = styled.div`
-  pointer-events: none;
-  position: absolute;
-  background-image: repeating-linear-gradient(
-    -45deg,
-    transparent 0 10px,
-    black 10px 20px
-  );
-  opacity: 0.05;
-  width: 100%;
-  height: calc((var(--fullbleed-height-px) - var(--page-height-px)) / 2);
-  z-index: 1;
-`;
 
-const PageAreaDangerZoneTop = styled(PageAreaDangerZone)`
-  top: 0;
-`;
-
-const PageAreaDangerZoneBottom = styled(PageAreaDangerZone)`
-  bottom: 0;
+  ${({ showSafeZone }) =>
+    showSafeZone &&
+    css`
+      &::before {
+        content: '';
+        width: 20px;
+        height: var(--page-height-px);
+        position: absolute;
+        border-top: 1px solid rgba(255, 255, 255, 0.4);
+        border-bottom: 1px solid rgba(255, 255, 255, 0.4);
+        left: -20px;
+      }
+
+      &::after {
+        content: '';
+        width: 20px;
+        height: var(--page-height-px);
+        position: absolute;
+        border-top: 1px solid rgba(255, 255, 255, 0.4);
+        border-bottom: 1px solid rgba(255, 255, 255, 0.4);
+        right: -20px;
+      }
+    `}
```

Comment:
Please consolidate shared properties into a single block as:

```suggestion
    css`
      &::before,
      &::after {
        content: '';
        width: 20px;
        height: var(--page-height-px);
        position: absolute;
        border-top: 1px solid rgba(255, 255, 255, 0.4);
        border-bottom: 1px solid rgba(255, 255, 255, 0.4);
      }
      &::before {
        left: -20px;
      }
      &::after {
        right: -20px;
      }
    `}
```